src/styles/shared/links.scss

```diff
@@ -8,27 +8,5 @@
 }
 
 @mixin underlined-link() {
```

Comment:
This mixin is used in two places, and now it's a single obvious property. What do you think to killing off the mixin and just using `text-decoration:underline;` directly where it's used in the Link and Button components to save a level of indirection?